internal/install/recipe_installer_test.go

```diff
@@ -390,17 +402,21 @@ func TestInstallAdvancedMode_bounce_on_enter(t *testing.T) {
 
 	v = validation.NewMockRecipeValidator()
 	p = &ux.MockPrompter{
-		PromptYesNoVal: true,
+		PromptYesNoVal:       true,
+		PromptMultiSelectVal: []string{testRecipeName},
 	}
 
 	i := RecipeInstaller{ic, d, l, f, e, v, ff, status, p, s}
 	err := i.Install()
 	require.NoError(t, err)
-	require.Equal(t, 2, statusReporters[0].(*execution.MockStatusReporter).RecipeSkippedCallCount)
```

Comment:
Since the `InstallContext` has `SkipLoggingInstall` set, I think we probably still want to test for marking at least that recipe skipped.  What do you think?

Reply:
👍